draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.stories.tsx

```diff
@@ -1,4 +1,4 @@
-import { Icon, Paragraph } from "@kaizen/component-library"
+import { Icon, Paragraph, Box } from "@kaizen/component-library"
```

Comment:
This story file was in the wrong place but I don't remember what that means.

Reply:
It means we're likely shipping the stories when we version + package this component 😓 Worth me fixing this now since I'm in the area.

Reply:
My bad 😬